src/test/java/net/imagej/ops/threshold/apply/LocalThresholdTest.java

```diff
@@ -769,4 +790,22 @@ public void testContingencyOfNormalImplementation() {
 		}
 	}
 
+	/**
+	 * Normalizes input image to [0.0, 1.0] range.
+	 */
+	private Img<DoubleType> normalizeImg(final Img<ByteType> input) {
```

Comment:
Good catch @dietzc! I have simplified the method in a recent commit.

The problem with `Ops.Image.Normalize` ops, however, is their restriction on the same type of input and output pixel types. Therefore, you still have to convert the input to the desired output type before applying those ops or am I missing something?